docs/UserGuide.md

```diff
@@ -19,10 +19,10 @@ Financial Assurance Revolutionary Telemarketer (FART) is a **desktop app for man
 
 2. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-F14b-4/tp/releases/tag/v1.4).
 
-3. Copy the file to the folder you want to use as the _home folder_ for your FART.
+3. Copy the file to the folder you want to use as the _home folder_ for your F.A.R.T.
    1. A good home folder would be the Downloads folder as it makes the next step easier.
 
-4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar fart_in_a.jar` command to run the application.<br>
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar F.A.R.T_in_a.jar` command to run the application.<br>
```

Comment:
Would including the fullstops make it hard for a user to type this command? Else, do remember to rename the jar file accordingly.